src/test/java/core/basesyntax/service/impl/CsvFileWriterTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CsvFileWriterTest {
+    private static final String VALID_FILE_TO_WRITE = "src/main/resources/output.csv";
+    private static final String WRONG_FILE_TO_WRITE = "2/2/2/output.csv";
+    private static final FileWriterService reportFileWriter = new CsvFileWriter();
+    private static final String EXPECTED_REPORT = """
+            fruit,quantity
+            banana,109
+            apple,109""";
+    private final List<String> expectedReaderResultAfterWrite = List.of(
+            "fruit,quantity", "banana,109", "apple,109");
+
+    @Test
+    void fileWriter_Ok_Equals() {
```

Comment:
Please read once again how to name test methods, by this name I don`t understand what is going on